Packs/CortexAttackSurfaceManagement/Scripts/InferWhetherServiceIsDev/InferWhetherServiceIsDev.py

```diff
@@ -217,13 +236,19 @@ def main():
         args = demisto.args()
```

Comment:
can you add the new hierarchy and provider args to the docstring for the `main` function? The reference to parameter `observed_key_value_pairs` might also need updating

Reply:
also this might be the more appropriate place to elaborate on what `hierarchy_info` is (as opposed to where i commented on the yml)

Reply:
[PR feedback](https://github.com/demisto/content/pull/30601/commits/cc59bddcbcfddffbc2e558a72a565654b0bfa5d1)